tests/integration_test/integration_test.sh

```diff
@@ -174,7 +196,7 @@ else
 fi
 
 function get_unused_port {
-    PORT=$(shuf -i 2000-65000 -n 1)
+    PORT=$(gshuf -i 2000-65000 -n 1)
```

Comment:
Should this be changed back to `shuf`?

Reply:
Darn. Yes. I really want to get rid of both of these since shuf works on Linux and gshuf works on Mac. Very frustrating...

Reply:
Yeah I always have to change it when running this locally 😞

Reply:
Removed it altogether by using RANDOM in bash.